### PR TITLE
VIH-7867 Fixed edit booking judge email field display

### PR DESCRIPTION
**Pull request by khizarkhanu-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-7867

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ export class AssignJudgeComponent extends BookingBaseComponent implements OnInit
     }
 
     get displayEmailField(): boolean {
-        return !!this.judge && this.judge.is_courtroom_account;
+        return !!this.judge && !ParticipantModel.IsEmailEjud(this.judge.email);
```
Comment (ZJianWang) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts:211]: can you add the new method IsEmailEjud to the test

Comment (khizarkhanu-hmcts) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/booking/assign-judge/assign-judge.component.ts:211]: added tests to test static method returning as expected.

File: AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts [hunk 2]
```diff
@@ -46,4 +46,8 @@ export class ParticipantModel {
               }
             : null;
     }
+
+    static IsEmailEjud(email: string): boolean {
+        return email && email.toLowerCase().includes('judiciary');
```
Comment (ZJianWang) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts:51]: I'm just considering is that better if we can use regex validate Ejud email here?

Comment (khizarkhanu-hmcts) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant.model.ts:51]: Not sure if that makes a difference, Regex for pattern match & validation, This code is to identify if email belongs to AAD judge or ejud. 
server side code is also on similar lines in HearingDetailsResponseExtensions
private static bool IsEmailEjud(string email)
{
    return !string.IsNullOrEmpty(email)  && email.Contains("judiciary", StringComparison.CurrentCultureIgnoreCase);
}

**Review by ZJianWang — APPROVED** [review]

**Pull request merged**